ui/shared/tabs/failureSummary/SuggestionsListItem.jsx

```diff
@@ -122,7 +129,12 @@ export default class SuggestionsListItem extends React.Component {
               rel="noopener noreferrer"
               title="Go to this line in the log viewer"
             >
-              <span className="align-middle">{suggestion.search} </span>
+              <span
+                className="align-middle link-style"
+                style={{ fontWeight: '500' }}
```

Comment:
If this is something we're gonna use often we could just make a custom class for it :)

Reply:
> oh, something odd I just noticed is that if you hover over a red task button the text goes white. Is that a reactstrap bug?

It's an artifact of the conditional outline based on `isSelected`.  So I added a custom `:hover` class to handle it.

Reply:
ah ok